archey/test/entries/test_archey_disk.py

```diff
@@ -17,62 +16,247 @@ def setUp(self):
         self.disk_instance_mock = HelperMethods.entry_mock(Disk)
         self.output_mock = MagicMock()
 
-    def test_disk_get_local_filesystems(self):
+    @patch(  # Prevent `_replace_apfs_volumes_by_their_containers` call (see next test).
+        "archey.entries.disk.platform.system",
+        return_value="Linux",
+    )
+    def test_disk_get_local_filesystems(self, _):
```

Comment:
How would you feel about changing the patch to mock the `_replace_apfs...` call rather than mocking the platform? This may save changing the test in future if we add more platform-dependent code but is slightly less readable

```suggestion
    @patch.object(  # Prevent `_replace_apfs_volumes_by_their_containers` call (see next test).
        Disk,
        "_replace_apfs_volumes_by_their_containers",
    )
    def test_disk_get_local_filesystems(self, apfs_disk_dict_mock):
```